packages/terra-consumer-nav/src/NavPropShapes.js

```diff
@@ -6,14 +6,18 @@ const navItemShape = {
    */
   text: PropTypes.string.isRequired,
   /**
-   * The path the profile link would redirect to.
+   * The path the link would redirect to.
    */
-  uri: PropTypes.string.isRequired,
+  url: PropTypes.string,
```

Comment:
oops, removed this when i was debugging a weird issue, will put it back, thanks for pointing it out!